strawberryfields/gbs/sample.py

```diff
@@ -335,7 +134,7 @@ def to_subgraphs(graph: nx.Graph, samples: list) -> list:
     graph_nodes = list(graph.nodes)
     node_number = len(graph_nodes)
 
-    subgraph_samples = [modes_from_counts(s) for s in samples]
+    subgraph_samples = [list(set(modes_from_counts(s))) for s in samples]
```

Comment:
Is the set necessary here?

Reply:
Yep see the comment above, `set` appears to have been controversial! 😆 
Generally I'm feeling we should have used `set` instead of `list` for our method of communicating subgraphs, but too late now!